File: JavaScript/Sortings/InsertionSort.js
const prompt = require('prompt-sync')({sigint:true})

const arr = [];
const n = parseInt(prompt("Enter number of elements: "));

console.log("Enter elements into the Array ")
for(i=0;i<n;i++){
    const a = parseInt(prompt());
    arr.push(a);
}

for(i=1;i<n;i++){
    let j = i ;
    while( j && (arr[j-1]>arr[j])){
        const temp = arr[j];
        arr[j] = arr[j-1];
        arr[j-1] = temp;
        j--;
    } 
}
console.log("Elements sorted using Insetion : "+arr);

File: C++ Language/Sortings/InsertionSort.cpp
#include<iostream>
using namespace std;

int main(void){
    int n;
    int i,j;
    int temp;
    cout << "Enter no of terms: ";
    cin >> n;
    int arr[n];
    cout << "Enter elements into the Array " << endl ;
    for(i=0;i<n;i++){
        cin >> arr[i];
    }
    cout << "Input taken " << endl;
    for(i=1;i<n;i++){
        j=i;
        while( j>0 && (arr[j]<arr[j-1])){ // change the '<' <=>  '>'
            temp = arr[j-1];
            arr[j-1] = arr[j];
            arr[j] = temp ;
            j--;
        }
    }
    cout << "Elements sorted using Insertion Algorithem: ";
    for(i=0;i<n;i++){
        cout << arr[i] << " ";
    }
}